src/cljs/rems/collapsible.cljs

```diff
@@ -41,15 +44,16 @@
   `:id` unique id required
   `:class` optional class for wrapper div
   `:open?` should the collapsible be open? Default false
+  `:dispatch-on-open` triggers the reframe event dispatch given as an argument when load-more is clicked
```

Comment:
Perhaps a regular callback function here (that in this case dispatches a re-frame event) would be a better API (not tied fo re-frame)?